tofhir-server/src/main/scala/io/tofhir/server/service/JobService.scala

```diff
@@ -2,23 +2,20 @@ package io.tofhir.server.service
 
 import com.typesafe.scalalogging.LazyLogging
 import io.tofhir.engine.model.FhirMappingJob
-import io.tofhir.server.model.JobMetadata
 import io.tofhir.server.service.job.{IJobRepository, JobFolderRepository}
 import io.tofhir.server.service.project.{IProjectRepository, ProjectFolderRepository}
 
 import scala.concurrent.Future;
 
-class JobService(jobRepositoryFolderPath: String, projectRepository: IProjectRepository) extends LazyLogging {
-
-  private val jobRepository: IJobRepository = new JobFolderRepository(jobRepositoryFolderPath, projectRepository.asInstanceOf[ProjectFolderRepository])
+class JobService(jobRepositoryFolderPath: String, jobRepository: IJobRepository, projectRepository: IProjectRepository) extends LazyLogging {
```

Comment:
**jobRepositoryFolderPath** and **projectRepository** are unused, we can remove them.